File: praktikum/praktikum-1/1-BunchOfKeys/BunchOfKeys.cpp
#include "BunchOfKeys.hpp"

// ctor
BunchOfKeys::BunchOfKeys(){
    n_keys = 0;
}

// member function
void BunchOfKeys::add(){
    n_keys++;
}

void BunchOfKeys::shake(){
    if(n_keys > 1){
        for(int i = 0; i < n_keys; i++){
            cout << "krincing" << endl;
        }
    } else{
        cout << "Tidak terjadi apa-apa" << endl;
    }
    
}

File: praktikum/praktikum-4/1-BunchOfKeys/BunchOfKeys.java
public class BunchOfKeys {
    private int n_keys;

    public BunchOfKeys(){
        n_keys = 0;
    }

    public void add(){
        n_keys++;
    }
    
    public void shake(){
        if(n_keys <= 1){
            System.out.println("Tidak terjadi apa-apa");
        
        } else{
            for(int i = 0; i < n_keys; i++){
                System.out.println("krincing");
            }
        }
    }
}
